docs/configuration.md

```diff
@@ -35,8 +35,20 @@ val conf = new SparkConf()
 val sc = new SparkContext(conf)
 {% endhighlight %}
 
-Note that we can have more than 1 thread in local mode, and in cases like spark streaming, we may actually
-require one to prevent any sort of starvation issues.
+Note that we can have more than 1 thread in local mode, and in cases like Spark Streaming, we may 
+actually require one to prevent any sort of starvation issues.
+
+Properties that specify some time duration should be configured with a unit of time. 
+The following format is accepted:
+ 
+    5s (seconds)
+    25ms (milliseconds)
```

Comment:
should these two be switched? Otherwise the ordering here isn't from smallest to largest